examples/chain/24_MsgRegisterAsDMM/example.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+
+	exchangetypes "github.com/InjectiveLabs/sdk-go/chain/exchange/types"
+	chainclient "github.com/InjectiveLabs/sdk-go/client/chain"
+	"github.com/InjectiveLabs/sdk-go/client/common"
+)
+
+func main() {
+	// network := common.LoadNetwork("mainnet", "k8s")
+	network := common.LoadNetwork("testnet", "k8s")
+	tmRPC, err := rpchttp.New(network.TmEndpoint, "/websocket")
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	senderAddress, cosmosKeyring, err := chainclient.InitCosmosKeyring(
+		os.Getenv("HOME")+"/.injectived",
+		"injectived",
+		"file",
+		"inj-user",
+		"12345678",
+		"5d386fbdbf11f1141010f81a46b40f94887367562bd33b452bbaa6ce1cd1381e", // keyring will be used if pk not provided
+		false,
+	)
+
+	if err != nil {
+		panic(err)
+	}
+
+	clientCtx, err := chainclient.NewClientContext(
+		network.ChainId,
+		senderAddress.String(),
+		cosmosKeyring,
+	)
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	clientCtx.WithNodeURI(network.TmEndpoint)
+	clientCtx = clientCtx.WithClient(tmRPC)
+
+	msg := &exchangetypes.MsgRegisterAsDMM{
+		Sender: senderAddress.String(),
+		DmmAccount: senderAddress.String(),
+	}
+
+	chainClient, err := chainclient.NewChainClient(
+		clientCtx,
+		network.ChainGrpcEndpoint,
+		common.OptionTLSCert(network.ChainTlsCert),
+		common.OptionGasPrices("500000000inj"),
+	)
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	for i := 0; i < 1; i++ {
```

Comment:
You can remove the for loop here so users are not confused. It was there because in the initial examples, I sent multiple messages with same content to test batch broadcasting feature.

Reply:
Done